README.md

```diff
@@ -30,6 +30,7 @@ API_LIMIT=500
 LOG_LEVEL=info
 ---------------------------------------------------
 CHANGE_VERSION_FILEPATH=<path_to_file>
+MAX_CORE=<your_machine's_processors_number> >>If this is necessary<<
```

Comment:
Using the machines number of processors is a good default for the maximum number of parallel calls. However, it may be helpful for the parameter name to be more generic than this. In some cases, I have found that the performance actually gets worse if I use that many parallel calls.

Suggestion:

1. Rename the config parameter so that it better describes _what_ is being set
2. In the readme, continue to mention that this number should probably be less than the machine's number of processors.